.drone.yml

```diff
@@ -9,23 +9,29 @@ clone:
     tags: true
 
 pipeline:
+  pre-build:
+    image: webhippie/nodejs:edge
```

Comment:
`edge` doesn't exist, just use `latest` which is based on regular alpine edge packaging.